Updating librdkafka (2.2.0 to 2.3.0) can present build error

Building against 2.3.0 on intel mac 

```
In file included from kfk.c:6:
/usr/local/include/librdkafka/rdkafka.h:8205:1: error: 'const' type qualifier on return type has no effect [-Werror,-Wignored-qualifiers]
const int rd_kafka_TopicPartitionInfo_partition(
^~~~~~
1 error generated.
```

No such error when building against 2.2.0

--- a/Makefile
+++ b/Makefile
@@ -7,7 +7,7 @@ else
         KFK_ROOT="${KAFKA_ROOT}"
 endif
 KFK_INCLUDE    = ${KFK_ROOT}/include
-W_OPTS         = -Wall -Wno-strict-aliasing -Wno-parentheses -Wextra -Werror -Wsign-compare
+W_OPTS         = -Wall -Wno-strict-aliasing -Wno-parentheses -Wextra -Werror -Wsign-compare -Wno-ignored-qualifiers
 OPTS           = -DKXVER=3 -shared -fPIC $(W_OPTS)
 LD_COMMON      = -lz -lpthread -lssl -g -O2
 LDOPTS_DYNAMIC = -L${KFK_ROOT}/lib/ -lrdkafka

